File: lib/Karve/CustomControls/DataGrid/Clases/DataGridColumnGroup/DataGridColumnGroup.vb
Imports Telerik.WinControls.UI

Public Class DataGridColumnGroup
    Inherits GridViewColumnGroup

    Public Sub Add(ByRef col As GridViewDataColumn)
        Me.Rows(0).Columns.Add(col)
    End Sub

    Public Sub New()
        Me.Rows.Add(New GridViewColumnGroupRow())
    End Sub
End Class

File: src/Controls/KarveControls/DataGrid/KarveGridTest/KarveGrid/ColumnGroup/DataGridColumnGroup.cs
using Telerik.WinControls.UI;

namespace KarveGrid.ColumnGroup
{
    public class DataGridColumnGroup : GridViewColumnGroup
    {

        public void Add(ref GridViewDataColumn col)
        {
            this.Rows(0).Columns.Add(col);
        }

        public DataGridColumnGroup()
        {
            this.Rows.Add(new GridViewColumnGroupRow());
        }
    }
}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
